content/docs/command-reference/metrics/diff.md

```diff
@@ -61,6 +60,8 @@ They're calculated between two commits (hash, branch, tag, or any
   path is allowed. It will override `xpath` defined in the corresponding
   DVC-file. See `dvc metrics show` for more details.
 
+- `--all` - list all metrics, even those without differences to compare.
```

Comment:
those that haven't changed? (hard to understand what does the option mean)

Reply:
Updated